new upstream releases of sci-biology/{aghermann,cnrun}

hmmr opened the pull request:
also shorten descriptions per repoman suggestion

marbre commented:
Did you use 'repo-commit' or 'repoman commit' for your changes? I think the updates Manifests are missing.

hmmr commented:
It's been long time since I pushed my last update here. Sorry. Will fix.

jlec commented:
>   digest.missing [fatal]        2

>   /home/travis/build/gentoo-science/sci/travis-overlay/sci-biology/aghermann::aghermann-1.0.7.tar.xz

>   /home/travis/build/gentoo-science/sci/travis-overlay/sci-biology/cnrun::cnrun-2.0.3.tar.xz

Please fix this by running repoman on those packages.

marbre commented on sci-biology/aghermann/Manifest:
```diff
@@ -1,3 +1,2 @@
-DIST aghermann-1.0.4.tar.xz 499872 SHA256 522a8c571c9e8aaecfb997f057a01298f804bff139c052d15b3347bdf2a58722 SHA512 ff891f71480ae25cce884bfea714c11eb0536cccfe4ade8c9b742d74db63dba6664e80d7f53272e95de1e04e1fdfb895ef278106a31a1d904e95cb062c4277c8 WHIRLPOOL fbfc6c3084b6d46f460b387cba521726b6fea4f4cacfd7907011507db66cc86700c7e36a1b6823d7a7fbd04d06986ec2e12654048544730765516265799af2f9
-DIST aghermann-1.0.5.tar.xz 500112 SHA256 a0c6ccd2e740f283559b92ee6fa7ff5c92699bae72cf10a7a7f6fb53e45a258e SHA512 3355664dfa59d81c110e5da4b883c09729fee440328b6c5b568f2b412d1ad3e09503832acfe4874a7b45f3330446bb2e3e144607e3af922c6ce2484162d6d4ab WHIRLPOOL ec8f5662345a9f67d59dfe7e80b8396eefcaba45e4be8ddd899e842ed35a617d92ea503d3380868f5e851a3b90d06018ecae6ad986bf015aee4f8aa14c89a211
+DIST aghermann-1.0.7.tar.xz 509336 SHA256 892b7cbb4a19554a772779e861a9f01b1bf753605722afb70a6ab362bebb2751 SHA512 bec591865a305ef3888c2f2986c3417139a853a9d5b95894c7b512fe48a220e40199a2c4ba84fa79c960e666e33b1f3371fbdc46c163d699a2041d0ba67eaebd WHIRLPOOL bcd02dcf06ac91f62b61014f6dace95d7721f0ba5e95e4080547e9c0a8b1a461fe3691da4360c8c33ab7b4ba04a35bcb6a0f006fcb4f9df5f918403ee5587627
+EBUILD aghermann-1.0.7.ebuild 667 SHA256 5ce4e22d937751368f976347b0313f0807c4e26f5b5713456df506bdcb51d5d7 SHA512 ab7d88839d2da8b32b903225f37898c1b0232fa74739823fe1c84e1a2fbf72b445de6627bd1df8dddb9ff3b2f46de3251037e23d4a78476bf274770382d5ec29 WHIRLPOOL 2434980c9fc21185b15e40986ead581007e372fc895f122ba50b4e7a720ddf9063d0706df250691cc165d14bdfe0bee2bd6e0ff85f32ad542070e3cdab695b65
```
It shouldn't be necessary to add an Manifest entry for the EBUILD, as the science overlay uses thin manifests. Don't know what went wrong here.

marbre commented on sci-biology/cnrun/Manifest:
```diff
@@ -1 +1,3 @@
-DIST cnrun-2.0.2.tar.xz 378672 SHA256 608437e014dca941df0f3c229f88ab6560e187392bef8170695ca9adc5bc2520 SHA512 e2eab04c29b2529c97b048907aaadc3b928357023b4873d82323baf70076abee74ba04c76e4bd3fbbc40dd98d571d1cd722c5d3bf9cde6c68bdcc7255f4b62fe WHIRLPOOL 3dd17dce302c9d0e3bb93577a39a97d5fa56816518e1518f49cb96c365a86621ee90ebc7311a6017416b814ce0b9205116627977b54b29b162e5c9ce6d25ea3e
+DIST cnrun-2.0.3.tar.xz 383368 SHA256 c6bc6859685782a245aa0aaa8bc98e3ef40da0b3d334440a59b1ba0493a37927 SHA512 b195be69802cdc2e6fa3601a045f5aac8c14c79d744c6d4dc383430236101d0db6c830d82b44be4e88f7f67f84c58e1c085cf9999673d451bff96778a1f3d938 WHIRLPOOL 30ab488a127c2d00c038f7b78a1661ce4653386e7a50d4ed0eb4bc6160fe62f4c8d1d6ace232dda59f48443fbfe72190ecaecb5b1cf50919d9e4269ebdcf5ef9
+EBUILD cnrun-2.0.3.ebuild 511 SHA256 2899d6a74edac104e78bb0d0acf3af6c62db95f0da55a8d7687e8b0c3014c64c SHA512 fe790cb454cd7736c5bda5e867e2869952c6e92531de0cd13feb46c145c16fa7ff1dc3c1a348f6075d8180c462cb5b004cd6a6858d05e2497e323565914051bb WHIRLPOOL 08318fa64ad53d5592778831c4b338d19ce976bbfe5ab82bffc792ea7faa6e76462d7bccc17875ccd4a9253f6e5a31714cd5f44ad9ce5c312eab7401031115e3
+MISC metadata.xml 1235 SHA256 c09a62acc2e8d1c9e2024f8000814ed84659fbb2ff521277cc1a4ca8b4e7612f SHA512 158948d0bff41030403d18afa615d40b1d47758017eabcc95bb84ac9c138109828cf079b7ff78983018209282bd4f4d5cdc219db0933bb5c4af01457039fb92e WHIRLPOOL dc6e11a61c38be0f595cac2970e2d9ff1aee774b9b316d5930bb13dac18024c3c7f6db7635217ba5c8be9e98126d9c17a1f2d99f20c8e8944a3833cbedba491e
```
As above. DIGESt for EBUILD and MISC shouldn't be present.

hmmr commented:
I'm a bit confused: when run in /usr/local/portage/sci-biology/aghermann, repoman says:

```
!!! A file is not listed in the Manifest: '/usr/local/portage/sci-biology/aghermann/aghermann-1.0.7.ebuild'
 * Missing digest for '/usr/local/portage/sci-biology/aghermann/aghermann-1.0.7.ebuild'
  ebuild.syntax [fatal]         1
   sci-biology/aghermann/aghermann-1.0.7.ebuild
  manifest.bad [fatal]          1
   sci-biology/aghermann/Manifest
Please fix these important QA issues first.
```

Regardless, I have removed non-DIST entries from Metadata in both packages, rebased and force-pushed.

marbre commented:
@hmmr I assume your local overlay doesn't uses thin manifest, but the science overlay does as configured https://github.com/gentoo-science/sci/blob/master/metadata/layout.conf#L32. You can use thin manifests for your local overlay too if preferred.
Further, you should check with repoman within your fork of the science overlay and not in your local overlay.

hmmr commented:
@marbre Indeed, the default (I didn't configure my local overlay at all) is to not use thin manifests, and that is the reason.

Thanks for bearing with me and helping me get things in order.
